### PR TITLE
Refactor and use babel

### PR DESCRIPTION
**Pull request by mamal72**
Hey. I've refactored the project and used Babel for transpiling it down to node v4.
I also did some minor changes on the tests to make the reason of leaking more clear.

Let me know if you have any problem with the presets or plugins or anything else. 👍

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    ["env", {
```
Comment (andywer) [inline .babelrc:3]: +1 for babel-preset-env :)

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+      "targets": {
+        "node": "4.0"
+      }
+    }]
+  ],
+  "plugins": [
+    "add-module-exports",
```
Comment (andywer) [inline .babelrc]: I don't think we need `add-module-exports`, since we only have named exports on package-level.

File: .babelrc [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "transform-es2015-modules-commonjs"
```
Comment (andywer) [inline .babelrc]: `transform-es2015-modules-commonjs`: Shouldn't that already be done by `babel-preset-env` + `node 4.0` target?

File: package.json [hunk 4]
```diff
@@ -2,9 +2,10 @@
   "name": "leakage",
   "version": "0.2.0",
   "description": "Memory leak testing for node. Write tests using your favorite test runner.",
-  "main": "lib/index.js",
+  "main": "build/index.js",
   "scripts": {
-    "test": "standard lib/**/*.js && mocha test/"
+    "build": "babel lib -d build",
```
Comment (andywer) [inline package.json]: I think it's more common to have a `src/` and a `lib/` directory. What do you say?

**Review by andywer — COMMENTED** [review]
Left a few minor questions and took the liberty two push two little commits to your PR.

File: .babelrc (hunk 2 above)
Comment (mamal72) [reply .babelrc]: Yup. It was for our `index.js` file. I've changed this file to avoid using this plugin. 👍

File: .babelrc (hunk 3 above)
Comment (mamal72) [reply .babelrc]: You are right. There was a problem with my NPM and dependencies and it wasn't working for me. Fixed. 👍

File: package.json (hunk 4 above)
Comment (mamal72) [reply package.json]: `src` with `lib`, `dist` or `build` are common names.
Changed to `src` and `lib`. Thanks! 🙏

**Review by andywer — APPROVED** [review]
Looks fine to me :)

**Pull request merged**